app.js

```diff
@@ -2,13 +2,16 @@ const express = require('express');
 const app = express();
 const {
   getTopics,
+  getEndpoints,
   handleNonExist
-} = require('./controllers/topics-controller')
+} = require('./controllers/controllers')
 
 app.use(express.json());
```

Comment:
Make sure not to include code you aren't using (in this case `app.use(express.json())`